### ISSUE TITLE
CA fit - TypeError: No loop matching the specified signature and casting was found for ufunc true_divide

### ISSUE THREAD

**Issue by adamcwatts** [MaxHalford/prince#96]
I get a TypeError using CA or MCA from the examples posted in the repository

```
pd.set_option('display.float_format', lambda x: '{:.6f}'.format(x))
X = pd.DataFrame(
    data=[
    [326, 38, 241, 110, 3],
    [688, 116, 584, 188, 4],
    [343, 84, 909, 412, 26],
    [98, 48, 403, 681, 85]
    ],
    columns=pd.Series(['Fair', 'Red', 'Medium', 'Dark', 'Black']),
    index=pd.Series(['Blue', 'Light', 'Medium', 'Dark'])
)

ca = prince.CA( n_components=2, n_iter=3, copy=True, check_input=True, engine='auto', random_state=42)
X.columns.rename('Hair color', inplace=True)
X.index.rename('Eye color', inplace=True)
ca = ca.fit(X)
```

```
TypeError                                 Traceback (most recent call last)
<ipython-input-25-6248e8f8c83f> in <module>
      3 X.index.rename('Eye color', inplace=True)
      4 X
----> 5 ca = ca.fit(X)

~\anaconda3\lib\site-packages\prince\ca.py in fit(self, X, y)
     43 
     44         # Compute the correspondence matrix which contains the relative frequencies
---> 45         X /= np.sum(X)
     46 
     47         # Compute row and column masses

TypeError: No loop matching the specified signature and casting was found for ufunc true_divide
```

**Comment by naadvar** [comment]
Hey, is there a fix for this error as  I still get it when using MCA on columns having all categorical variables?

**Comment by SammieH21** [comment]
I had to make a change to the source code such that X /= np.sum(X) became X = X / np.sum(X) in the ca.py script. Not fun, I know. Did you get the same error when installing from pip?

**Comment by MaxHalford** [comment]
You know you can make a pull request if you've corrected a bug :). I don't have time to look into this properly at the moment.

**Comment by naadvar** [comment]
@SammieH21 I tried changing it to  X /= np.sum(X) . I still get the same issue.

**Comment by SammieH21** [comment]
Do you mean that you tried changing it to X = X / np.sum(X)?

**Comment by naadvar** [comment]
@SammieH21 yes. Sorry. I changed it to that I still get the same error

**Comment by MaxHalford** [comment]
This should be fixed with version 0.7.1.

**Comment by SammieH21** [comment]
Awesome, and thank you!
